hoomd/mpcd/pytest/test_update.py

```diff
@@ -0,0 +1,200 @@
+# Copyright (c) 2009-2024 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+import pytest
+import hoomd
+import numpy as np
+
+from hoomd.conftest import pickling_check
+from hoomd.logging import LoggerCategories
+from hoomd.conftest import logging_check
+
+
+@pytest.fixture
+def snap():
+    snap = hoomd.Snapshot()
+    if snap.communicator.rank == 0:
+        snap.configuration.box = [20, 20, 20, 0, 0, 0]
+        snap.particles.N = 0
+        snap.particles.types = ["A"]
+        snap.mpcd.types = ["A"]
+        snap.mpcd.mass = 1.0
+    return snap
+
+
+@pytest.fixture
+def flow():
+    flow = hoomd.mpcd.update.ReverseNonequilibriumShearFlow(trigger=1,
+                                                            num_swaps=1,
+                                                            slab_width=1.0,
+                                                            target_momentum=2.0)
+    return flow
+
+
+decompositions = [None]
+if hoomd.communicator.Communicator().num_ranks > 1:
```

Comment:
This should be `==` so that it only inserts them when the right number of ranks are available
```suggestion
if hoomd.communicator.Communicator().num_ranks == 2:
```